.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: bash
```

Comment:
Cannot find "bash" in the language documentation of travis-ci. I would have expected c/cpp here